app/src/main/java/com/example/ubclaunchpad/launchnote/addPhoto/PhotoInfoActivity.kt

```diff
@@ -0,0 +1,78 @@
+package com.example.ubclaunchpad.launchnote.addPhoto
+
+import android.os.Bundle
+import android.support.v7.app.AppCompatActivity
+import android.view.View
+import butterknife.ButterKnife
+import butterknife.OnClick
+import com.example.ubclaunchpad.launchnote.R
+import com.example.ubclaunchpad.launchnote.database.LaunchNoteDatabase
+import com.example.ubclaunchpad.launchnote.models.PicNote
+import io.reactivex.Observable
+import io.reactivex.android.schedulers.AndroidSchedulers
+import io.reactivex.schedulers.Schedulers
+
+import kotlinx.android.synthetic.main.activity_photo_info.*
+
+class PhotoInfoActivity : AppCompatActivity() {
+
+    private lateinit var picNoteToEdit: PicNote
+    private var removeIfNoSave: Boolean = false
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_photo_info)
+        ButterKnife.bind(this)
+
+        picNoteToEdit = intent.extras.getSerializable(PIC_NOTE_ARG) as PicNote
+        removeIfNoSave = intent.extras.containsKey(REMOVE_IMAGES_IF_NO_CHANGE) &&
+                intent.extras.getBoolean(REMOVE_IMAGES_IF_NO_CHANGE)
+
+        title_input.setText(picNoteToEdit.title)
+        description_input.setText(picNoteToEdit.description)
+        // todo vpineda we need to create a dropdown for all of the classes or projects!
+        class_input.setText(picNoteToEdit.classId.toString())

```

Comment:
could you check if it's still the default Id, and if it is, leave the text field blank so that they see the hint? That way it's clearer to the user that they're supposed to be typing in the class or project name.